torch/_inductor/select_algorithm.py

```diff
@@ -124,6 +125,9 @@ def __init__(
         self.epilogue_fn = epilogue_fn
         self.render_hooks = dict()
         self.triton_meta: Optional[Dict[str, object]] = None
+        # For Templated Attention
+        self.subgraphs = subgraphs
```

Comment:
Right now we are only supporting 1 "modification" we should update this and subgrapsh in tandem to be equal length and support multiple

Reply:
will leave as a todo in a future PR